Examples/example_inputs/minimal_DISCON.IN

```diff
@@ -92,8 +92,20 @@ Extra lines that ROSCO doesn't care about anymore
 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.00088027 0.01116187 0.02310060 0.03164156 0.03968639 0.04741131 0.05899071 0.07019329 0.08090119 0.09141661 0.10169361 0.11174124 0.12170066 0.13137494 0.14103377 0.15048120 0.15989580 0.16913227 0.17831010 0.18739638 0.19643591 0.20537011 0.21419674 0.22293719 0.23161365 0.24025975 0.24885012 0.25735704 0.26578040 0.27407178 0.28233051 0.29047594 0.29867838 0.30674517 0.31490089 0.32284411               ! PS_BldPitchMin    - Minimum blade pitch angles [rad]
 
 !------- SHUTDOWN -----------------------------------------------------------
-0.436300000000      ! SD_MaxPit         - Maximum blade pitch angle to initiate shutdown, [rad]
-0.418880000000      ! SD_CornerFreq     - Cutoff Frequency for first order low-pass filter for blade pitch angle, [rad/s]
+0                 ! SD_EnablePitch          - Shutdown when collective blade pitch exceeds a threshold, [-]
+0                 ! SD_EnableYawError        - Shutdown when yaw error exceeds a threshold, [-]
```

Comment:
Are these required inputs?  If not, can they be removed here and run without error?

Reply:
I want these to be optional inputs needed only when SD_Mode = 1. I'll check this and update.

Reply:
Resolved in f674adc